scripts/find_fixed_tickets.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
A header that explains what it does without having to look this ticket up through git praise would be nice.